examples/default/realign.json

```diff
@@ -0,0 +1,67 @@
+{
+  "@context": "https://github.com/Inria-Visages/blob/master/BIDS-prov/context.json",
+  "@id": "http://example.org/ds00000X",
+  "generatedAt": "2020-03-10T10:00:00",
+  "wasGeneratedBy": {
+      "@id": "INRIA",
+      "@type": "Project",
+      "startedAt": "2016-09-01T10:00:00",
+      "wasAssociatedWith": { "@id": "NIH",
+                              "@type": "Organization",
+                              "hadRole": "Funding"
+                            }
+    },
+    "exp_dir" : {
+      "@id" : "exp_dir",
+      "prov:atLocation" : "$HOME/BIDS-prov/nidm-results-examples/spm_default/ds011/"
+    },
+  "@graph": [
+    {
+      "prov:Activity" : [
```

Comment:
@cmaumet  I have added type indexing like you asked (I hope)
Can you confirm this is the way it should be done ?

Reply:
This looks good to me. You should be able to check this using the conversion from the json-ld playground.